app.php

```diff
@@ -0,0 +1,233 @@
+            ]
+        ]
+    );
+
+    $collectionID = $response['id'];
+
+    var_dump($response);
+}
+
+/**
+ * @throws Exception
+ */
+function list_collection()
+{
+    global $database;
+
+    echo str_cli_color("Running List Collection API", 'f_green, b_black');
+
+    $response = $database->listCollections();
+
+    $collection = $response['$collection'];
+
+    var_dump($collection);
+}
+
+/**
+ * @throws Exception
+ */
+function add_doc()
+{
+    global $collectionId;
+    global $database;
+
+    $collectionId = "0";
+
+    echo str_cli_color("Running Add Document API", 'f_green,b_black');
+
+    $response = $database->createDocument(
+        $collectionId,
+        [
+            'name' => "Spider Man",
+            'release_year' => '1920',
+        ],
+        ['*'],
+        ['*']
+    );
+
+    var_dump($response);
+}
+
+/**
+ * @throws Exception
+ */
+function upload_files()
+{
+    global $storage;
+    $filename = 'test.txt';
+
+    echo str_cli_color("Running upload file API", 'f_green,b_black');
+
+    $response = $storage->createFile(
+        curl_file_create($filename),
+        [],
+        []
+    );
+
+    var_dump($response);
+}
+
+/**
+ * @throws Exception
+ */
+function list_files()
+{
+    global $storage;
+
+    echo str_cli_color("Running List Files API", 'f_green,b_black');
+
+    $result = $storage->listFiles();
+
+    $file_count = $result['sum'];
+    $files = $result['files'];
+
+    var_dump($file_count, $files);
+}
+
+/**
+ * @throws Exception
+ */
+function delete_file()
+{
+    global $storage;
+
+
+    echo str_cli_color("Running Delete File API", 'f_green', 'b_black');
+
+    $result = $storage->listFiles();
+    $first_file_id = 'test.txt';
+    //$first_file_id = $result['files'][0]['$id'];
+    $response = $storage->deleteFile($first_file_id);
+
+    var_dump($response);
+}
+
+/**
+ * @param $email
+ * @param $password
+ * @param $name
+ * @throws Exception
+ */
+function create_user($email, $password, $name)
+{
+    global $userId, $users;
+
+    echo str_cli_color("Running create user API", 'f_green,b_black');
+
+    $response = $users->create(
+        $email,
+        $password,
+        $name
+    );
+    $userId = $response['$id'];
+
+    var_dump($response);
+}
+
+/**
+ * @throws Exception
+ */
+function list_user()
+{
+    global $users;
+
+    echo str_cli_color("Running list user api", "f_green,b_black");
+
+    $response = $users->list();
+
+    var_dump($response);
+}
+
+/**
+ * @throws Exception
+ */
+function run_all_tasks()
+{
+
+    $name = time();
+    create_collection();
+    list_collection();
+    add_doc();
+    //list_doc();
+    upload_files();
+    list_files();
+    delete_file();
+    create_user(
+        $name . '@test.com',
+        $name . '@123',
+        $name
+    );
+    list_user();
+}
+
+try {
+    run_all_tasks();
+} catch (Exception $e) {
+    die($e->getMessage());
+}
+
+echo str_cli_color("successfully run playground", "f_green,b_black");
```

Comment:
I personally don't like the requirement of a whole library just to colorize this string. Please print a normal string and remove the package as requirement.

Reply:
100% agree. The idea of this repo is to have a **very** simple example to showoff how easy it is to integrate and use the Appwrite SDK. We should avoid any un-required dependencies.

Reply:
A simple echo with a line break at the end should be just fine.

Reply:
Okay will do it

Reply:
removed the extra dependecies